src/GovUk.Education.ExploreEducationStatistics.Common/Model/Either.cs

```diff
@@ -96,6 +96,11 @@ public static class EitherExtensions
 
     public static class EitherTaskExtensions
     {
+        public static async Task<bool> IsRight<TFailure, TSuccess>(this Task<Either<TFailure, TSuccess>> task)
```

Comment:
Should we add the `IsLeft` variant of this as well?

Reply:
Added `IsLeft` variant now, albeit not used currently.